lib/kinde_sdk/kinde_client_sdk.ex

```diff
@@ -998,6 +998,51 @@ defmodule KindeClientSDK do
     end
   end
 
+
+  @doc """
+Initializes the connection with access and refresh tokens.
+
+## Parameters
+- `conn`: The connection to assign the tokens to.
+- `access_token`: The access token to be stored.
+- `refresh_token`: The refresh token to be stored.
+
+## Returns
+- The updated connection with assigned tokens.
+"""
+def init_with_tokens(conn, access_token, refresh_token) do
+
+    if is_nil(access_token) or is_nil(refresh_token) do
+      raise ArgumentError, "Tokens cannot be nil"
+    end
+
+    client = %{
+      access_token: access_token,
+      refresh_token: refresh_token
+    }
+
+    Conn.assign(conn, :kinde_client, client)
+
+end
+
+
+  def get_kinde_client(conn) do
+    conn.assigns[:kinde_client]
+  end
+
+
+  def authenticated_request(conn, method, url, params \\ %{}) do
```

Comment:
I'm not sure what this method is supposed to do?
It takes a `method`, `url` and `params` but doesn't do anything with it, and just returns `headers`?

Reply:
Its been resolved and pushed.